### ISSUE TITLE
The order of the parameters of `nn.Conv1d()`, `nn.Conv2d()` and `nn.Conv3d()` should be explained in the actual order of the parameters.

### ISSUE THREAD

**Issue by hyperkai** [pytorch/pytorch#135880]
### 📚 The doc issue

The docs of [nn.Conv1d()](https://pytorch.org/docs/stable/generated/torch.nn.Conv1d.html), [nn.Conv2d()](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html) and [Conv3d()](https://pytorch.org/docs/stable/generated/torch.nn.Conv3d.html) show parameters as shown below:

> CLASS torch.nn.Conv1d(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True, padding_mode='zeros', device=None, dtype=None)[[source]](https://pytorch.org/docs/stable/_modules/torch/nn/modules/conv.html#Conv1d)

> CLASS torch.nn.Conv2d(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True, padding_mode='zeros', device=None, dtype=None)[[source]](https://pytorch.org/docs/stable/_modules/torch/nn/modules/conv.html#Conv2d)

> CLASS torch.nn.Conv3d(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True, padding_mode='zeros', device=None, dtype=None)[[source]](https://pytorch.org/docs/stable/_modules/torch/nn/modules/conv.html#Conv3d)

But the order of the parameters to explain is not in the actual order of the parameters above as shown below. *`padding_mode` should be explained at the last: 

> Parameters
> - in_channels ([int](https://docs.python.org/3/library/functions.html#int)) – Number of channels in the input image
> - out_channels ([int](https://docs.python.org/3/library/functions.html#int)) – Number of channels produced by the convolution
> - kernel_size ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple)) – Size of the convolving kernel
> - stride ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – Stride of the convolution. Default: 1
> - padding ([int](https://docs.python.org/3/library/functions.html#int), [tuple](https://docs.python.org/3/library/stdtypes.html#tuple) or [str](https://docs.python.org/3/library/stdtypes.html#str), optional) – Padding added to both sides of the input. Default: 0
> - padding_mode ([str](https://docs.python.org/3/library/stdtypes.html#str), optional) – 'zeros', 'reflect', 'replicate' or 'circular'. Default: 'zeros'
> - dilation ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – Spacing between kernel elements. Default: 1
> - groups ([int](https://docs.python.org/3/library/functions.html#int), optional) – Number of blocked connections from input channels to output channels. Default: 1
> - bias ([bool](https://docs.python.org/3/library/functions.html#bool), optional) – If True, adds a learnable bias to the output. Default: True

### Suggest a potential alternative/fix

So, the order of the parameters to explain should be as shown below. *`padding_mode` is explained at the last:

> Parameters
> - in_channels ([int](https://docs.python.org/3/library/functions.html#int)) – Number of channels in the input image
> - out_channels ([int](https://docs.python.org/3/library/functions.html#int)) – Number of channels produced by the convolution
> - kernel_size ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple)) – Size of the convolving kernel
> - stride ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – Stride of the convolution. Default: 1
> - padding ([int](https://docs.python.org/3/library/functions.html#int), [tuple](https://docs.python.org/3/library/stdtypes.html#tuple) or [str](https://docs.python.org/3/library/stdtypes.html#str), optional) – Padding added to both sides of the input. Default: 0
> - dilation ([int](https://docs.python.org/3/library/functions.html#int) or [tuple](https://docs.python.org/3/library/stdtypes.html#tuple), optional) – Spacing between kernel elements. Default: 1
> - groups ([int](https://docs.python.org/3/library/functions.html#int), optional) – Number of blocked connections from input channels to output channels. Default: 1
> - bias ([bool](https://docs.python.org/3/library/functions.html#bool), optional) – If True, adds a learnable bias to the output. Default: True
> - padding_mode ([str](https://docs.python.org/3/library/stdtypes.html#str), optional) – 'zeros', 'reflect', 'replicate' or 'circular'. Default: 'zeros'

cc @svekars @brycebortree @tstatler @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

**Comment by malfet** [comment]
Please propose the fix to address the doc issue

**Comment by cmagapu** [comment]
@hyperkai I'm new to open source and would like to contribute. Please let me know if I can work on this.

**Comment by kit1980** [comment]
Looks like this was fixed by https://github.com/pytorch/pytorch/pull/134618
Cherry-picking that into 2.5
